lib/iris/util.py

```diff
@@ -862,6 +862,12 @@ def __cmp__(self, other):
             result = NotImplemented
         return result
 
+    def __lt__(self, other):
```

Comment:
As elsewhere, it'd be good to get rid of `__cmp__`. In both cases it'd be good to define the other rich-comparison methods as well, that way we can keep the behaviour the same.